packages/algob/sample-project/algob.config.js

```diff
@@ -31,6 +54,7 @@ let defaultCfg = {
 module.exports = {
   networks: {
     localhost: defaultCfg,
-    default: defaultCfg
+    default: defaultCfg,
+    KMD: KMDConfig
```

Comment:
You add KMDConfig as a network but it shouldn't be used as a node that can compile code.

Reply:
correct.